content/2016/04/behind-the-scenes-high-risk-shipping-addresses.md

```diff
@@ -20,6 +20,7 @@ review.
 This blog post investigates some high risk shipping addresses known to MaxMind,
 as well as provides some general fraud review tips for identifying them.
 
+<!--lint disable no-emphasis-as-heading-->
```

Comment:
I wonder if it would be better to fix these. It seems like we disabled it in a lot of places. I guess it is fine though.